.github/ISSUE_TEMPLATE/release.md

```diff
@@ -18,46 +18,43 @@ Please assign the release manager to the issue.
+- [ ] Checked that the [docs release issue] created previously has been completed.
+- [ ] Updated, in a single PR, the following on `master`:
+  - [ ] The [releases table] in the `README.md`, removing the now old unsupported release and adding the new one.
+  - [ ] The `baseBranches` list in `.github/renovate.json5`, removing the now old unsupported release and adding the new one.
+  - [ ] The [`crossplane/test` repo test workflows][crossplane-test-workflows] to ensure the checkout release branch and helm install version(s) point at the new release across all the workflow files.
 - [ ] Ensured that users have been notified of the release on all communication channels:
   - [ ] Slack
   - [ ] Twitter
-- [ ] Updated the [releases table] in the `README.md` on `master`.
-- [ ] Updated the current release version in the [Crossplane docs website repo].
-- [ ] Updated the release branch reaching EOL with [docs removal directive].
-- [ ] Request @jbw976 to remove the EOL docs version from Google Search
 
 
 <!-- Named Links -->
+[Code Freeze]: https://crossplane.io/docs/master/reference/release-cycle.html#code-freeze
+[ci-workflow]: https://github.com/crossplane/crossplane/actions/workflows/ci.yml
+[configurations-workflow]: https://github.com/crossplane/crossplane/actions/workflows/configurations.yml
+[crossplane-test-workflows]: https://github.com/crossplane/test/tree/master/.github/workflows
 [docs release issue]: https://github.com/crossplane/docs/issues/new?assignees=&labels=release&template=new_release.md&title=Release+Crossplane+version...+
-[releases.crossplane.io]: https://releases.crossplane.io
-[xpkg.upbound.io]: https://cloud.upbound.io/browse
 [new release]: https://github.com/crossplane/crossplane/releases/new
+[promote-workflow]: https://github.com/crossplane/crossplane/actions/workflows/promote.yml
+[release-1.11.0]: https://github.com/crossplane/crossplane/issues/3600
 [releases table]: https://github.com/crossplane/crossplane#releases
-[Crossplane docs website repo]: https://github.com/crossplane/docs
-[docs removal directive]: https://github.com/crossplane/crossplane/pull/3003
+[releases.crossplane.io]: https://releases.crossplane.io
 [tag-workflow]: https://github.com/crossplane/crossplane/actions/workflows/tag.yml
-[ci-workflow]: https://github.com/crossplane/crossplane/actions/workflows/ci.yml
-[configurations-workflow]: https://github.com/crossplane/crossplane/actions/workflows/configurations.yml
-[promote-workflow]: https://github.com/crossplane/crossplane/actions/workflows/promote.yml
-[crossplane-test-workflows]: https://github.com/crossplane/test/tree/master/.github/workflows
-[release-1.7]: https://github.com/crossplane/crossplane/issues/2977
-[Code Freeze]: https://crossplane.io/docs/master/reference/release-cycle.html#code-freeze
\ No newline at end of file
+[xpkg.upbound.io]: https://cloud.upbound.io/browse
```

Comment:
what do you think about updating this link to https://marketplace.upbound.io/configurations?query=getting-started, both here and in the patch release template?  this could be a more direct way to find the released configurations